templates/actors/npc-sheet-2.hbs

```diff
@@ -0,0 +1,511 @@
+                        </li>
+
+                        {{!-- Type --}}
+                        <li class="creature-type">
+                            {{#if editable}}
+                            <a class="config-button" data-action="type" data-tooltip="DND5E.CreatureTypeConfig"
+                               aria-label="{{ localize "DND5E.CreatureTypeConfig" }}">
+                                <span>{{ labels.type }}</span>
+                                <i class="fas fa-cog"></i>
+                            </a>
+                            {{else}}
+                            <span>{{ labels.type }}</span>
+                            {{/if}}
+                        </li>
+
+                        {{!-- Alignment --}}
+                        <li class="creature-alignment">
+                            {{#if editable}}
+                            <input type="text" name="system.details.alignment" value="{{ source.details.alignment }}"
+                                   placeholder="{{ localize "DND5E.Alignment" }}">
+                            {{else}}
+                            <span>{{ system.details.alignment }}</span>
+                            {{/if}}
+                        </li>
+
+                    </ul>
+
+                </div>
+
+                {{!-- Resting, Special Traits, & Proficiency --}}
+                {{#if actor.isOwner}}
+                <div class="right">
+
+                    <div class="left">
+
+                        <div class="sheet-header-buttons">
+                            <button type="button" class="short-rest gold-button" data-tooltip="DND5E.ShortRest"
+                                    aria-label="{{ localize "DND5E.ShortRest" }}">
+                                <i class="fas fa-utensils"></i>
+                            </button>
+                            <button type="button" class="long-rest gold-button" data-tooltip="DND5E.LongRest"
+                                    aria-label="{{ localize "DND5E.LongRest" }}">
+                                <i class="fas fa-campground"></i>
+                            </button>
+                            <button type="button" class="config-button gold-button" data-action="flags"
+                                    data-tooltip="DND5E.SpecialTraits"
+                                    aria-label="{{ localize "DND5E.SpecialTraits" }}">
+                                <i class="fas fa-star"></i>
+                            </button>
+                        </div>
+
+                        <div class="proficiency">
+                            <span>{{ localize "DND5E.Proficiency" }}</span>
+                            <strong>{{ dnd5e-formatModifier system.attributes.prof }}</strong>
+                        </div>
+
+                    </div>
+
+                    {{!-- CR --}}
+                    <div class="right cr-badge badge" aria-label="{{ localize "DND5E.CRLabel" cr=system.details.cr }}">
```

Comment:
It would be good to have a tooltip here, at least when not in editing mode, to give some indication of what this number means for people new to the sheet.